src/CommandExecutor.ts

```diff
@@ -51,6 +51,12 @@ class BuiltInCommandExecutionStrategy extends CommandExecutionStrategy {
 class SystemFileExecutionStrategy extends CommandExecutionStrategy {
     startExecution() {
         return new Promise((resolve, reject) => {
```

Comment:
The purpose of the strategy classes is to [replace conditions with polymorphism](http://www.refactoring.com/catalog/replaceConditionalWithPolymorphism.html). Please create a separate strategy for Windows.